src/main/java/core/basesyntax/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.db.Storage;

```

Comment:
The `BalanceOperation` class should be placed in its own package named `strategy` to follow the checklist requirement of keeping all logic in a single package.